Lib/sysconfig.py

```diff
@@ -546,6 +546,7 @@ def get_config_vars(*args):
 
         if os.name == 'nt':
             _init_non_posix(_CONFIG_VARS)
+            _CONFIG_VARS['TZPATH'] = ''
```

Comment:
I think we should be guaranteeing that it's set, and it should be an error condition otherwise. The "reasonable default" varies by platform, and can be configured at build time.

On Windows, the reasonable default is actually an empty string (no tzdata), but also I'd like to add it as a flag at build time.